WordPress/gutenberg
# Content Block: Change placeholder and e2e test to refer to Content block

## What?
Updates the placeholder text for the Content block (which was recently changed from the 'Post Content' block) so it no longer refers to _itself_ as the Post Content block.

## Why?
Recently the Post Content block was changed to just be the 'Content' block. This update brings the placeholder text up to date.

Addresses  https://github.com/WordPress/gutenberg/issues/53894

## Testing Instructions

1. Open or add a template in the site editor
2. Add a 'Content' block
3. Confirm that the placeholder content refers to the 'Content' block, not the 'Post Content' block

## Screenshots or screencast 
<img width="891" alt="Screenshot 2023-08-24 at 12 51 54 pm" src="https://github.com/WordPress/gutenberg/assets/1842363/2d3b59cd-7297-4b6e-9fdc-9f51d61dca33">
